aws/macie_types.go

```diff
@@ -0,0 +1,29 @@
+package aws
+
+import (
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/gruntwork-io/go-commons/errors"
+)
+
+type Macie struct {
+	AccountIds []string
+}
+
+func (r Macie) ResourceName() string {
+	return "macie"
```

Comment:
This should be `macie-member` to differentiate with `macie-admin`.

Reply:
Done in 2dfdc00